pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "aws-operator"
 	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "9.1.2-dev"
+	version            = "9.1.1"
```

Comment:
Why without any suffix?

Reply:
fixed, forgot to change back after testing